lib/resources/r4/charge_item.yaml

```diff
@@ -105,3 +105,17 @@ fields:
         - display: Bill Code Schedule
           system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/14002
           info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-14002-bill-code-schedule
+
+  - name: performer.function
```

Comment:
The structure we used for communication had the parent field and then the children with the terminology bindings. Do we want to do that structure here as well? https://github.com/cerner/fhir.cerner.com/blob/main/lib/resources/r4/related_person.yaml#L26

cc @alexbeckner1

Reply:
I'm fine with following the same pattern as what was done on relatedPerson.communication.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/682/commits/050eba2bc5d5c77b30288e3d85fb076e792e451f#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR109-R128

Reply:
Left some comments on the commit.

Reply:
responded here https://github.com/cerner/fhir.cerner.com/commit/050eba2bc5d5c77b30288e3d85fb076e792e451f#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR109-R128